### ISSUE TITLE
Provide relase.

### ISSUE THREAD

**Issue by martiniturbide** [kev009/boo2pdf#1]
Hi Kevin.

Can it be possible to provide a binary release of your converter? like a .jar that can be easly run with java.exe.
 
Also the online converter isn't working (http://ps-2.kev009.com/boo2pdf/). It gives me a "405 Not Allowed" error.

Thanks for your efforts on preserving old IBM documentation. I have interest on all OS/2 material available. 

Regards

**Comment by Fish-Git** [comment]
@kev009 

> Hi Kevin.
> 
> Can it be possible to provide a binary release of your converter? like a .jar that can be easly run with java.exe.
> 
> Also the online converter isn't working (http://ps-2.kev009.com/boo2pdf/). It gives me a "405 Not Allowed" error.
> 
> Thanks for your efforts on preserving old IBM documentation. I have interest on all OS/2 material available.
> 
> Regards

I agree with Martin. Your online converter throws a "405 Not Allowed" error.
